ripple/src/main/webapp/businessdetails.js

```diff
@@ -219,7 +256,291 @@ function calculatePrice(avgPriceRatings, numPriceRatings) {
   }
 }
 
-/* Display the number of dollar signs for price rating on the page */
+/* Add a review: Display the number of dollar signs for price rating on the page */
```

Comment:
not sure how i understand price is related to review. Was it intentional to add to all the comments?

Reply:
Yes it was intentional. When you add a review, you're rating by stars and by price

Reply:
Hmm actually, it's more for displaying a review summary of all users. Will change.